src/js/components/Pagination/Pagination.js

```diff
@@ -49,7 +49,11 @@ const Pagination = forwardRef(
       Math.min(page, totalPages) || 1,
     );
 
-    useEffect(() => setActivePage(page), [page]);
+    useEffect(() => {
+      setActivePage(page);
+      const pageEvent = new Event('paginationupdate');
```

Comment:
Small naming comment, wondering if calling the event "pagechange" would be more descriptive.